declarations/Grabo.bg.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Grabo.bg",
+  "documents": {
+    "General Conditions of Sale": {
```

Indeed, this is very clear:

```suggestion
    "Commercial Terms": {
```

From automated translation:

> This document is aimed at the commercial partners of "Grabo Media" AD, publishing offers on the information platform Grabo.bg. This document aims to comprehensively describe all principles and circumstances in the work of "Grabo Media" AD with its commercial partners, in order to guarantee maximum transparency in implementation of [Regulation (EU) 2019/1150 of the European Parliament and the Council of June 20, 2019. to promote fairness and transparency for business users of intermediary online services](https://translate.google.com/website?sl=auto&tl=en&hl=fr&client=webapp&u=https://eur-lex.europa.eu/legal-content/EN-BG/TXT/?uri%3DCELEX%253A32019R1150).

(spot-on for this instance, then 😉)